env.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/sh
 
 export BRAVE_TOR_VERSION="1"
-export TOR_VERSION="0.3.2.10"
+export TOR_VERSION="0.3.3.6"
+
+# Tor has not released the win32 expert bundle for 0.3.3.6.
+export TOR_WIN32_VERSION="0.3.2.10"
 export ZLIB_VERSION=1.2.11
 export LIBEVENT_VERSION=2.1.8-stable
 export OPENSSL_VERSION=1.0.2o
-export TOR_VERSION=0.3.2.10
 
 export ZLIB_HASH=c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
 export LIBEVENT_HASH=965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2
 export OPENSSL_HASH=ec3f5c9714ba0fd45cb4e087301eb1336c317e0d20b575a125050470e8089e4d
-export TOR_HASH=60df77c31dcf94fdd686c8ca8c34f3b70243b33a7344ecc0b719d5ca2617cbee
+export TOR_HASH=99bc59f6dbf395894de12f3a83b3251a82dfd93dc7f6d3afcbbd80f6111433b7
 
 export LIBEVENT_KEY=9E3AC83A27974B84D1B3401DB86086848EF8686D
 export TOR_KEY=2133BC600AB133E1D826D173FE43009C4607B1FB
+
+export KEYSERVER=hkp://ipv4.pool.sks-keyservers.net:80
```

Comment:
can we change this to one of the keyservers that supports HKPS? for instance  hkps://hkps.pool.sks-keyservers.net

(i found this in https://sks-keyservers.net/overview-of-pools.php)